cmd/destroy.go

```diff
@@ -205,13 +207,12 @@ func destroyLab(ctx context.Context, c *clab.CLab) (err error) {
 						log.Debugf("Worker %d terminating...", i)
 						return
 					}
-					name := cont.ID
-					if len(cont.Names) > 0 {
-						name = strings.TrimLeft(cont.Names[0], "/")
-					}
-					err := c.Runtime.DeleteContainer(ctx, name)
+					//if len(cont.Names) > 0 {
```

Comment:
shall we bring this back? to display which container we might filed to delete?

Reply:
yes we can do that. I'm not sure anymore why it was deleted

Reply:
brought back in 0bde6cf